lottie/src/main/java/com/airbnb/lottie/LottieAnimationView.java

```diff
@@ -504,6 +506,13 @@ public void setComposition(@NonNull LottieComposition composition) {
     setImageDrawable(lottieDrawable);
 
     requestLayout();
+
+    for (LottieOnCompositionLoadedListener lottieOnCompositionLoadedListener : lottieOnCompositionLoadedListeners) {
+      if(lottieOnCompositionLoadedListener != null) {
```

Comment:
This should never happen. `addCompositionLoadedListener` isn't marked as `@Nullable`. If you want to make it extra safe, you can do a preconditions check in `addLottieOnCompositionLoadedListener` but don't just silently ignore nulls here